code/modules/mob/living/carbon/human/life/handle_orders.dm

```diff
@@ -0,0 +1,38 @@
+/mob/living/carbon/human/proc/handle_orders()
+
+	var/aura_center = null
+	if(current_aura)
+		aura_center = src
+
+	if((src.job == JOB_SQUAD_LEADER || HAS_TRAIT(src, TRAIT_SOURCE_SQUAD_LEADER)) && src.assigned_squad && src.assigned_squad.num_tl) //If the guy giving orders is leading a squad with FTLs we need them to act as beacons
+		for(var/mob/living/carbon/human/marine in src.assigned_squad.ftl_list)
+			marine.handle_ftl_orders(marine)
+
+	if(aura_strength > 0)
+		for(var/mob/living/carbon/human/H as anything in GLOB.alive_human_list)
```

Comment:
this is really expensive to call on every human every life() tick

it's more acceptable for xenoes bc they're rarely dealing with 30+ xenoes, but humans regularly breach 100, along with a not-exactly-free get_dist check